package.json

```diff
@@ -61,7 +61,7 @@
     "ember-popper-modifier": "^2.0.0 || ^3.0.0 || ^4.0.0",
     "ember-ref-bucket": "^4.0.0 || ^5.0.0",
     "ember-render-helpers": "^0.2.0",
-    "ember-style-modifier": "^0.8.0 || ^1.0.0 || ^2.0.0 || ^3.0.0",
+    "ember-style-modifier": "^0.8.0 || ^1.0.0 || ^2.0.0 || ^3.0.0 || ^4.0.0",
```

Comment:
Do we need to still support the older versions?

Reply:
`ember-style-modifier` is used by several addons. Often apps depend on it indirectly through more as one addon. If each of them only support few versions it can get hard or impossible to find a compatible combination. Easy to end up in dependency hell.

Supporting all these versions is not much overhead for us. The API uses in Ember Bootstrap hasn't changed through these versions.